### ISSUE TITLE
Spam on Dedicated Server

### ISSUE THREAD

**Issue by Wubzzz1337** [Cycro54/Revive_Me#2]
It legit just spams this for a lot:

```
[21Mar2023 20:36:08.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:08.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:09.781] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:09.781] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:09.781] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:11.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:11.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:11.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:12.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:12.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:12.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:12.538] [Server thread/INFO] [com.dynamic_view.DynView/]: Mean tick: 11ms increasing chunk view distance to: 13
[21Mar2023 20:36:12.551] [Server thread/DEBUG] [Smooth Boot (Reloaded)/]: Initialized IO-Worker-18
[21Mar2023 20:36:12.779] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:12.779] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:12.779] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:13.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:13.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:13.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:13.784] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:13.784] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:13.784] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:13.886] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:13.886] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:13.886] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:14.185] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:14.185] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:14.185] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:14.787] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:14.787] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:14.787] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:14.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:14.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:14.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:15.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:15.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:15.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:15.782] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:15.782] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:15.782] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:15.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:15.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:15.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:16.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:16.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:16.183] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:16.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:16.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:16.783] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:16.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:16.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:16.883] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:17.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:17.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:17.182] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:17.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:17.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:17.882] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:18.885] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:18.885] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:18.885] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:33.184] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:33.184] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:33.184] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:33.979] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:33.979] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:33.979] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
[21Mar2023 20:36:39.432] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE EVENT CANCELLED? false
[21Mar2023 20:36:39.432] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS IT CLIENTSIDE? false
[21Mar2023 20:36:39.433] [Server thread/INFO] [invoker54.reviveme.common.event.attackFallenEvents/]: IS THE ATTACKED ENTITY A PLAYER? false
```

**Comment by Cycro54** [comment]
Honestly thought I silenced all of those. That'll be dealt with as soon as I get a chance, hopefully today. Ty for telling meh

**Comment by Wubzzz1337** [comment]
Side question, were you thinking about also making this Fabric compatible? :p

**Comment by Cycro54** [comment]
Fixed in the latest versions for mc 1.18 and 1.19

As for making it Fabric compatible, that is a huge maybe!
But it will certainly be a while
